docs/iris/example_code/General/polynomial_fit.py

```diff
@@ -0,0 +1,51 @@
+"""
+Fitting a polynomial
+====================
+
+This example demonstrates adding a polynomial fit to a 1D plot of data from
```

Comment:
>Also the plot doesn't work anything like as well unless the fit is added to the cube's metadata!

Well, it works just fine if you plot the fit AuxCoord against the time coord of the cube, you don't need to add the AuxCoord to the cube to achieve this (but I agree that you should in this case, to carry the new data in a context where it means something).